### PR TITLE
JMX Receiver added

### PR DESCRIPTION
**Pull request by ppradyu**
Added JMX receiver for Cassandra integration

### CODE REVIEW

File: pkg/agent/hostagent.go
Comment (bhogayatakb) [inline pkg/agent/hostagent.go:1]: On the factory portion jmx is required so thats okay.
Lets Use the naming Cassandra everywhere else.

+ We can improvise the indentation as well

Comment (ppradyu) [reply pkg/agent/hostagent.go:1]: >  Lets Use the naming Cassandra everywhere else.

So I tried adding cassandra at everyplace one by one but that was breaking one thing or the other. So kept JMX at all places. Cassandra can be added at config level but not currently here.

>  We can improvise the indentation as well

Added commit for same. This occurred due to conflict resolution with master branch.

**Review by bhogayatakb — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 **/node_modules
-otelstruct.go
\ No newline at end of file
+otelstruct.go
+build/*
```
Comment (parth-dadhaniya) [inline .gitignore:3]: Add new line character,

Comment (ppradyu) [reply .gitignore]: Done

**Review by parth-dadhaniya — APPROVED** [review]

**Pull request merged**